config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-presubmit-1.25.yaml

```diff
@@ -45,7 +45,7 @@ presubmits:
       extra_refs:
         - org: kubernetes-sigs
           repo: cluster-api-provider-azure
-          base_ref: release-1.7
+          base_ref: release-1.8
```

Comment:
No test run since this change:

- https://testgrid.k8s.io/provider-azure-cloud-provider-azure-1-25-presubmit#pr-cloud-provider-azure-e2e-ccm-vmss-capz-1-25

Reminder to check back later!